index.html

```diff
@@ -2,5 +2,5 @@
 # You don't need to edit this file, it's empty on purpose.
 # Edit theme's home layout instead if you wanna make some changes
 # See: https://jekyllrb.com/docs/themes/#overriding-theme-defaults
-layout: home
----
+layout: paginated
+---
```

Comment:
- Pagination does not work on `index.md`, instead, we should use `index.html`.
- I uses a dedicated layout "`paginated`" but we could override "`home`".